packages/compiler/lib/REFDOC.md

```diff
@@ -0,0 +1,497 @@
+## `@format`
```

Comment:
The md file format looks fine to me. Let's generate md files for other artifact types.